Airports/EGKK/SMR/Regions.txt

```diff
@@ -925,13 +983,16 @@ N051.09.12.407 W000.10.19.387
 N051.09.12.350 W000.10.18.552
 N051.09.12.352 W000.10.17.520
 N051.09.12.395 W000.10.16.514
-;KA_Island
+; - KA_Island
```

Comment:
See previous comments again, this is adding a notch into the east side of the island alongside KA.